### PR TITLE
Changelog in English

### PR DESCRIPTION
**Pull request by Grimimi**
I deleted the old changelog (<= 1.0.0) which was not very useful.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,86 +2,63 @@
 
 ## Release (1.3.0)
 
-### Ajouts généraux 
-### Commandes ajoutées
+In this release, we decided to use English as the main language.
 
-### Corrections de bogues
+### additions
```
Comment (acs-l) [inline CHANGELOG.md]: Personally, I would remove the section's header (`additions`, which should be `Additions` by the way) if there are no entries for this peculiar release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,86 +2,63 @@
...
-
-### Commandes ajoutées
-
-* `camera_scrolling?` (@xvw, reporté par @Zer0Zer0x)
-* `windowskin_tone` et `windowskin_opacity` (@xvw, reporté par @YendaHusk)
-* `event_tone` et `player_tone` (@xvw, reporté par @Zer0Zer0x)
-* `event_character_name`, (@BastienDuplessier)
-* `event_character_index`, (@BastienDuplessier)
-* `event_change_character`, (@BastienDuplessier)
-*    - `menu_enabled?|disabled?`
```
Comment (acs-l) [inline CHANGELOG.md]: The leading bullet's symbol (`*`) should be removed. Could you remove the **trailing whitespace** as well ?

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,86 +2,63 @@
...
-*  Correction des commandes `armors_possessed` et `weapons_possessed` (@hinola)
-*  Corrections des commandes `event_flash` et `player_flash` (@BastienDuplessier)
-*  Correction de l'afficheur de texte (profil avec contours) (@xvw reporté par @jauke)
-*  Correction des interpolations linéaires (@xvw reporté par @hinola)
-
+### additions
+*  Add command `last_used_item`, (@hinola, reported by @Zer0Zer0x)
+*  Add command `disable_weather_dimness`+ `disable_weather_dimness` (@BastienDuplessier, reported by @Zer0Zer0x)
+*  Add command `window_exists?` (@hinola)
+*    - Add command `camera_locked?`
```
Comment (acs-l) [inline CHANGELOG.md]: The leading bullet's symbol (`*`) should be removed. Could you remove the **trailing whitespace** as well ? [#sameplayershootagaint]

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,86 +2,63 @@
...
-
+### additions
+*  Add command `last_used_item`, (@hinola, reported by @Zer0Zer0x)
+*  Add command `disable_weather_dimness`+ `disable_weather_dimness` (@BastienDuplessier, reported by @Zer0Zer0x)
+*  Add command `window_exists?` (@hinola)
+*    - Add command `camera_locked?` 
+     - Add command `camera_lock_x`
+     - Add command `camera_lock_y`
+     - Add command `camera_unlock_x`
+     - Add command `camera_unlock_y`
```
Comment (acs-l) [inline CHANGELOG.md]: Could you remove the **trailing whitespace** ?

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,86 +2,63 @@
...
+*  Add command `window_exists?` (@hinola)
+*    - Add command `camera_locked?` 
+     - Add command `camera_lock_x`
+     - Add command `camera_lock_y`
+     - Add command `camera_unlock_x`
+     - Add command `camera_unlock_y` 
+     (@BastienDuplessier, reported by @lunri)
+*  Add command `mouse_hover_window?` (@hinola)
+*  Add commands `choice` and `last_choice` (@xvw, reported by @Zer0Zer0x)
+*  - Add command `dash_activation`
```
Comment (acs-l) [inline CHANGELOG.md]: The sub bullets' symbol (`-`) should be removed: the sublist should be flatten.
Could you remove the **trailing whitespace** as well ? [#sameplayershootagain]

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,86 +2,63 @@
...
-* Correction de la commande `event_trigger` (@xvw)
+### additions
+* Add range support in the syntax shortcuts for variables : `SV[1, 2, 1..18] = 10` (for example) (@xvw)
+* Add command `camera_scrolling?` (@xvw, reported by @Zer0Zer0x)
+* Add commandsn `windowskin_tone` and `windowskin_opacity` (@xvw, reported by @YendaHusk)
+* Add commands `event_tone` and `player_tone` (@xvw, reported by @Zer0Zer0x)
+* Add command `event_character_name` (@BastienDuplessier)
+* Add command `event_character_index` (@BastienDuplessier)
+* Add command `event_change_character` (@BastienDuplessier)
+*    - Add command `menu_enabled?|disabled?`
```
Comment (acs-l) [inline CHANGELOG.md]: The sub bullets' symbol (`-`) should be removed: the sublist should be flatten.
Could you remove the **trailing whitespace** as well ? [#sameplayershootagain]

**Comment by acs-l** [comment]
In my opinion, the syntax/format used in this `CHANGELOG.md` is lacking on many aspects. Aspects which are covered in the following standard [***keepachangelog***](http://keepachangelog.com/en/1.0.0/).
Let's just cover a few:
- Dating *Releases*
- Using an `Unreleased` section for issues which are not dispatched in a specific release yet
- Specifying the relevant sections such as the ones suggested:
    - `Added` for new features (*commands*)
    - `Changed` for changes in existing features (*commands*)
    - `Deprecated` for soon-to-be-removed features
    - `Removed` for now removed features
    - `Fixed` for any bug fixes


Additionnally, I think that it would be suitable to have a link between "*Commands' addition*", "*Commands' update*", "*Bug fixes*" and their related issues.
For instance:

```markdown
## Unreleased

### Bug fixes

| `Command` | Description | Related issue |
| --------- | ----------- | ------------- |
| `move_to` | Pathfinding issues | #268 |
```

---

(The branch should not be prefixed as a `feature/` one as it does not add any value to the project itself. Prefixing it with `refactoring/` or `project-management/` would be more convenient; altough that's only my little personal point of view 😅.)

**Comment by xvw** [comment]
> Additionnally, I think that it would be suitable to have a link between "Commands' addition", "Commands' update", "Bug fixes" and their related issues.

Is that really necessary ?

**Comment by BastienDuplessier** [comment]
I don't think this is the place to discuss about changing the changelog.
This is only about translating what's already there.

Let's create an issue about those changelog best practice!

**Comment by acs-l** [comment]
Bastien is right, I am moving this discussion in the *contributors* group ;).

**Comment by BastienDuplessier** [comment]
Don't close without merging ;)

**Pull request merged**